pkg/cmd/opts/install.go

```diff
@@ -344,10 +344,7 @@ func (o *CommonOptions) InstallHelm() error {
 	if err != nil || !flag {
 		return err
 	}
-	latestVersion, err := util.GetLatestVersionFromGitHub("kubernetes", "helm")
-	if err != nil {
-		return err
-	}
+	latestVersion := "v2.16.1"
```

Comment:
Yeah, this needs to use the version stream

Reply:
How can this be achieved?